canbench-bin/src/csv_file.rs

```diff
@@ -2,8 +2,9 @@ use canbench_rs::{BenchResult, Measurement};
 use std::{collections::BTreeMap, fs::File, io::Write, path::PathBuf};
 
 /// Delimiter used in the CSV file.
-/// Tab `\t` avoids issues with commas in numbers or text and is reliably parsed by Google Sheets.
-const DELIMITER: char = '\t';
+/// Use `,` for GitHub/VSCode preview.
+/// Use `\t` for better compatibility with Google Sheets.
```

Comment:
Why doesn't google sheets handle the comma? I wouldn't expect there are commas in the numbers or text anyway.

Reply:
google sheets only reconginze tab automatically. all the other delimiters a pasted into a single cell and you need to do extra manual clicks to get them recognized. this is strange, especially considering that 'comma' is in the CSV abbreviation. idk why they do it like that. maybe internationalisation issues.